result/isislsp.yaml

```diff
@@ -0,0 +1,341 @@
+components:
+  schemas:
+    IsisLsps.State.Request:
+      description: >-
+        The request to retrieve BGP peer prefix information.
+      type: object
+      properties:
+        isis_router_names:
+          description: >-
+            The names of ISIS Routers to return results for.
+            An empty list will return results for all ISIS router.
+          type: array
+          items:
+            type: string
+          x-constraint:
+            - "/components/schemas/Device.IsisRouter/properties/name"
+        lsp_filters:
+          description: >-
+            Specify which LSPs to return. If the list is empty or missing
+            then all LSPs will be returned, e.g. '6400000000010000'.
+          type: array
+          items:
+            type: string
+            format: hex
+  
+    IsisLsps.State:
+      description: >-
+        ISIS LSP.
+      type: object
+      properties:
+        isis_router_name:
+          description: >-
+            The name of a ISIS Router.
```

Comment:
Grammar : name of a ISIS Router --> name of the ISIS Router.

Second, is this referring to the name that is configured on our side or the learnt name?

Reply:
This is name of ISIS the routers configured in our side that got learned LSP information.

Reply:
"name of a ISIS Router" --> "name of the ISIS Router"

Reply:
done